charts_common/lib/src/chart/common/behavior/selection/select_nearest.dart

```diff
@@ -134,7 +134,11 @@ class SelectNearest<D> implements ChartBehavior<D> {
       case SelectionTrigger.longPressHold:
         _listener = GestureListener(
             onTapTest: _onTapTest,
-            onTap: _onCustomTap,
+            onTap: hoverEventDelay == null
+                ? _onCustomTap
```

Comment:
this was also in my earlier throttling test. it doesn't quite work with this code. gotta remember to check my git status before branching.